content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md

```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
+
+- Burp Suite Pro/Community
+- Postman
+- OWASP Zed Attack Proxy
+{{% /expand %}}
+
+{{% expand "Automated web crawling and web scanner configuration tweaking" %}}
+</br>
+
+Our pentesters use both commercial and freeware security tools to assess the targeted
+application. They'll modify these tools as needed, to make sure that scanning can find
+security issues on every segment of your asset, and the application as a whole.
+
+In addition, our pentesters run automated crawls to:
+
+- Identify any pages are available to unauthenticated users
+- Determine the full site tree.
+
+Our pentesters may use web crawling and scanning tools such as:
+
+- Nmap
+- Burp Suite Pro/Community
+- Nikto
+{{% /expand %}}
+
+{{% expand "Authenticated vulnerability scanning / Manual crawling" %}}
+</br>
+
+In this part of the pentest process, our pentesters:
+
+- Use automated tools for web application crawling
+  - Verify the results manually
+- Run manual crawling tests for better coverage
+  - Verify authentication on protected areas of the application
+
+With automated scanning, our pentesters
+
+- Assess the application using the authenticated sessions where applicable
+
+Our pentesters use extreme caution to minimize impact on the targeted system.
+They may use vulnerability scanning tools such as:
+
+- WPScan
+- Burp Suite Pro/Community
+- sqlmap
+
+{{% /expand %}}
+
+{{% expand "Manual web vulnerability tests / exploit reviews / microservices" %}}
+</br>
+
+Cobalt pentesters use tool-assisted manual tests to identify and analyze the
+following parts of the app for vulnerabilities:
+
+- Functionality
+- Business logic
+- Deployment
+
+The assessment identifies published vulnerabilities, including those listed in the 
+
+- OWASP Top 10
+- CVE reports or tracked by CVE entries
+
+Our pentesters also consider the workflows and business logic into consideration
+when they identify vulnerabilities in the application.
+
+The assessment includes tests for vulnerabilities such as:
+
+- Injection attacks that probe the robustness of server-validation routines
+- Session management flaws that could allow user impersonation
+- Flaws in access control that expose data or enable users to gain elevated privileges.
+
+If the application includes microservices, our pentesters focus on interactions
+between different systems. They examine the implementation of:
+
+- Access control management
+- Cross-Origin Resource Sharing (CORS)
+
+We thoroughly examine:
+
+- Access control management
+  - Cross-Origin Resource Sharing (CORS) implementation
+- Vulnerabilities outlined in the OWASP [API Security Project](https://owasp.org/www-project-api-security/)
+
+For each finding, pentesters determine the risk associated with each issue by:
+
+- Demonstrate how the issue could be exploited
+- Evaluate its impact within the context of the business function, data, and
+  users of the asset
+- Set up a Proof-of-Concept exploitation to:
```

Comment:
```suggestion
- Demonstrating how the issue could be exploited
- Evaluating its impact within the context of the business function, data, and
  users of the asset
- Setting up a Proof-of-Concept exploitation to:
```

Reply:
Yeah, you're right, in context, we should go with the "gerund"